crates/ruff_python_whitespace/src/whitespace.rs

```diff
@@ -0,0 +1,15 @@
+/// Returns `true` for [whitespace](https://docs.python.org/3/reference/lexical_analysis.html#whitespace-between-tokens)
+/// characters.
+pub const fn is_python_whitespace(c: char) -> bool {
+    matches!(
+        c,
+        // Space, tab, or form-feed
+        ' ' | '\t' | '\x0C'
```

Comment:
This used to include `\r` and `\n`, but it turns out all of the call sites matched on those separately anyway.